internal/db/migration_callbacks.go

```diff
@@ -0,0 +1,33 @@
+package db
+
+import (
+	"context"
+)
+
+type migrationCallback func(ctx context.Context) error
+
+var migrationCallbacks = make(map[int32]migrationCallback)
+
+func init() {
+	migrationCallbacks[13] = updateFingerprint
+}
+
+func HasCallback(seq int32) bool {
+	_, ok := migrationCallbacks[seq]
+	return ok
+}
+
+func CallCallback(ctx context.Context, seq int32) error {
+	if cb, ok := migrationCallbacks[seq]; ok {
+		return cb(ctx)
+	}
+
+	return nil
+}
+
+func updateFingerprint(ctx context.Context) error {
+	// TODO creates a cyclic dependency - migration code must be moved outside of db
+	//pkd := dao.GetServiceDao(ctx)
+	//return pkd.RecalculatePubkeyFingerprints(ctx)
+	return nil
+}
```

Comment:
The idea is to create internal/migrations/go and internal/migrations/sql packages where both SQL and Go code would live.